include/boost/geometry/algorithms/union.hpp

```diff
@@ -139,14 +113,13 @@ struct union_insert
 template
 <
     typename Linear1, typename Linear2, typename LineStringOut,
-    bool Reverse1, bool Reverse2, bool ReverseOut
+    typename TagIn1, typename TagIn2
```

Comment:
Where are the Reverse parameters now? (I can probably look that up but in a review screen that is not so clear)